packages/sku/lib/runESLint.js

```diff
@@ -3,6 +3,7 @@ const { yellow, cyan, gray } = require('chalk');
 const { ESLint } = require('eslint');
 const eslintConfig = require('../config/eslint/eslintConfig');
 const { lintExtensions } = require('./lint');
+const assert = require('node:assert');
```

Comment:
What do you think about adding an ESLint rule to require the `node:` protocol?

Reply:
That's a good idea. Had to manually update a bunch of them. Do we go with [the `eslint-plugin-unicorn` version](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/6d15a02d48de7ecfc38d0683a8487b2f937d83a0/docs/rules/prefer-node-protocol.md) or [a standalone one](https://www.npmjs.com/package/eslint-plugin-node-import)?

Reply:
Let's go with our rainbow friend 👍

Reply:
Will do it in a separate PR for ease-of-review.